src/pages/steps/PlaceStep.tsx

```diff
@@ -106,6 +106,16 @@ const PlaceStep = ({
     );
   };
 
+  const filterByCallback = (place: Place, props) => {
+    const name = getPlaceName(place.name, place.mainLanguage);
+    const address = getAddress(place.address, place.mainLanguage);
+
+    return (
+      address.streetAddress.toLowerCase().indexOf(props.text.toLowerCase()) !==
+        -1 || name.toLowerCase().indexOf(props.text.toLowerCase()) !== -1
```

Comment:
```suggestion
      address.streetAddress.toLowerCase().includes(props.text.toLowerCase()) || 
      name.toLowerCase().includes(props.text.toLowerCase())
```

You can probably write it like this right?

Reply:
Yes ;)